data/ecosystem/vendors.yaml

```diff
@@ -356,3 +356,10 @@
   contact: 'https://github.com/hagen1778'
   oss: true
   commercial: true
+- name: Tracetest
+  distribution: false
+  nativeOTLP: true
+  url: 'https://tracetest.io/'
```

Comment:
This should point to your documentation where OTLP is mentioned and used to point my traces to